Dockerfile

```diff
@@ -2,4 +2,4 @@ FROM docker.io/microlib/golang-base:1.7
 
 # This will change depending on each microservice entry point
 # This will change to the binary start in production based images
-CMD ["/bin/sh", "go run main.go"]
+CMD ["/usr/local/go/bin/go", "run", "main.go"]
```

Comment:
@luigizuccarelli I was wondering if we shouldn't add an `ENTRYPOINT` that simply runs the `go` executable. And then the `CMD` on top of that. In that way we might not need to change the Dockerfile for production, we just run it with a different command.

What do you think?